javascript-3/script3.js

```diff
@@ -0,0 +1,48 @@
+const marca = ["BMW", "Audi", "Dacia", "Audi", "Audi", "Dacia", "Dacia", "Dacia", "BMW", "Volkswagen", "Opel", "Mercedes", "BMW", "Volkswagen"];
+const model = ["Seria 1", "A3", "Spring", "Q5", "A4", "Spring", "Docker", "Logan", "X5", "Golf", "Astra", "E 200", "220i", "Golf"];
+const pretCumparare = [16000, 9000, 17000, 20300, 6000, 8000, 10000, 6000, 35500, 18900, 41000, 22500, 20500, 11590];
+let celMaiMicPret = pretCumparare[0];
+let celMaiMarePret = 0;
+let celeMaiIeftineMasini = new Array();
+let celeMaiScumpeMasini = new Array();
+let text = "";
+let text2 = "";
+
+for (let i = 0; i < pretCumparare.length; i++) {
+    if (celMaiMicPret > pretCumparare[i]) {
+        celMaiMicPret = pretCumparare[i];
+    }
+
+    if (celMaiMarePret < pretCumparare[i]) {
+        celMaiMarePret = pretCumparare[i];
+    }
+}
+
+for (let i = 0; i < pretCumparare.length; i++) {
+    if (pretCumparare[i] === celMaiMicPret) {
```

Comment:
Super! Ai tratat si cazul in care exista mai multe masini cu pretul minim si maxim